### PR TITLE
Streamlined version of make_unattended.sh

### PR DESCRIPTION
**Pull request by e-minguez**
Fabian told me it wasn't needed all that hustle but instead just append another cpio as an _overlay_

### CODE REVIEW

File: slemicro/make_unattended.sh [hunk 1]
```diff
@@ -67,91 +67,113 @@ TMPDIR="$(mktemp -d -t make_unattended.XXXXXX)"
...
-
-# Now we need to compress the filesystem
-zstd -q "${TMPDIR}"/second
-rm -f "${TMPDIR}"/second
-
-# And create the proper initrd by concatenating both
-cat "${TMPDIR}"/first "${TMPDIR}"/second.zst > "${TMPDIR}"/initrd
-rm -f "${TMPDIR}"/first "${TMPDIR}"/second.zst
+# Finally, we append the .profile into the initrd as:
+echo "${TMPDIR}"/.profile | cpio -o -F "${TMPDIR}"/initrd 2>/dev/null
```
Comment (Vogtinator) [inline slemicro/make_unattended.sh:89]: I suggest to use `--quiet` instead of `2>/dev/null` to not hide error messages.

Comment (Vogtinator) [inline slemicro/make_unattended.sh:89]: Does `-F "${TMPDIR}"/initrd` append?

Comment (Vogtinator) [inline slemicro/make_unattended.sh:89]: Is this written to the right location inside the archive? It might end up at `${TMPDIR}` inside the initrd

Comment (e-minguez) [reply slemicro/make_unattended.sh]: Good suggestion, thanks!

Comment (e-minguez) [reply slemicro/make_unattended.sh]: Actually, I'm not sure, switching back to good old >> :) Thanks!

**Review by atanasdinov — APPROVED** [review]

**Pull request merged**